src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+class RegistrationServiceTest {
+    private static  RegistrationServiceImpl registrationService;
+    private static StorageDaoImpl storageDao;
+    private static final User user = new User();
+    private static final User actual = new User();
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        user.setLogin("PerfectLogin");
+        user.setPassword("Perfect!94?Password");
+        user.setAge(19);
+        storageDao.add(user);
+    }
+
+    @BeforeEach
+    void setUp() {
+        actual.setLogin("OtherLogin");
+        actual.setPassword("OtherPassword");
+        actual.setAge(20);
+    }
+
+    @Test
+    public void registerPositiveTest() {
+        Assertions.assertNotNull(registrationService.register(actual));
+        Storage.people.clear();
+        storageDao.add(user);
+    }
+
+    @Test
+    public void nullUserTest() {
```

Comment:
rename tests according to convention <methodUnderTest>_<state>_<expectedBehavior>